$NetBSD: patch-CMakeLists.txt,v 1.3 2021/07/12 18:47:15 adam Exp $

LLD 7.0.1 is incompatible with gtest version in pkgsrc.  Remove
the condition that prevents it from building its own copy when system
version is installed.

--- CMakeLists.txt.orig	2021-04-06 16:38:18.000000000 +0000
+++ CMakeLists.txt
@@ -76,7 +76,6 @@ if(CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRE
       endif()
       set(UNITTEST_DIR ${LLVM_MAIN_SRC_DIR}/utils/unittest)
       if(EXISTS ${UNITTEST_DIR}/googletest/include/gtest/gtest.h
-          AND NOT EXISTS ${LLVM_LIBRARY_DIR}/${CMAKE_STATIC_LIBRARY_PREFIX}gtest${CMAKE_STATIC_LIBRARY_SUFFIX}
           AND EXISTS ${UNITTEST_DIR}/CMakeLists.txt)
         add_subdirectory(${UNITTEST_DIR} utils/unittest)
       endif()
